server/commands/src/impls.rs

```diff
@@ -126,3 +127,61 @@ fn update_gamemode(ctx: &mut CommandCtx, gamemode: Gamemode, entity: Entity) {
         ctx.game.handle(&mut *ctx.world, event);
     }
 }
+
+#[command(usage = "tell|msg|w <target> <message>")]
+pub fn whisper(
+    ctx: &mut CommandCtx,
+    target: EntitySelector,
+    message: TextArgument,
+) -> anyhow::Result<()> {
+    let sender_name = if let Some(sender_name) = ctx.world.try_get::<Name>(ctx.sender) {
+        sender_name.0.clone()
+    } else {
+        panic!("`Name` component not found for whisper command sender");
```

Comment:
Would the name `Server` be appropriate since (as far as I can tell, at the moment) a /tell not generated by a player would be from the Server or something running in the server?

Reply:
Yes, that's about right.